integration.sh

```diff
@@ -75,6 +75,11 @@ function run_tests() {
 	else
 		sleep 1s
 		go test -tags "cassandra gocql_debug" -timeout=5m -race $args
+
+		ccm clear
+		ccm start
+		sleep 1s
```

Comment:
it seels 1s is not enough for some Travis tests to pass. Maybe try to up it to 10s?

Reply:
Sure we can try that.

Reply:
Could we add a loop (with inner sleep maybe 1s or 100ms) that checks whether the database is ready instead? Not relying on a constant sleep should make it more reliablie and responsive (if we need to wait less than those 10 seconds)

Reply:
We can absolutely do that @martin-sucha but it's more work and needs more testing so perhaps in another PR. What do you guys think @alourie ?

Reply:
This seems to work as well @alourie thx for the suggestion. What do you think @martin-sucha this should be good enough. I will run it a few more times and then push an update.

Reply:
Pushed a new version using the `--wait-for-binary-proto` flag instead @alourie

Reply:
Looks good to me! :+1: